src/xtgeo/__init__.py

```diff
@@ -97,6 +97,7 @@ def _xprint(msg):
     GridProperties,
     gridproperties_dataframe,
     gridproperties_from_file,
+    list_properties,
```

Comment:
It may be that `list_properties` is too generic since this is actually grid properties. I think `list_gridproperties` will be more precise

Reply:
Good idea, I have updated it.